source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -217,6 +224,68 @@ def wrapper(*args, **kwargs):
         return wrapper
 
 
+# Add TAD deep links in each section of type on_demand_transport if network contains app_code in codes
+class add_tad_links(object):
+    def __call__(self, f):
+        @wraps(f)
+        def wrapper(*args, **kwargs):
+            objects = f(*args, **kwargs)
+            if objects[1] != 200:
+                return objects
```

Comment:
Can we have this in a separate function please (`has_invalid_response_code`) that can be shared across all journey decorators ? 
It would be easier to read/understand, and if the response ever changes (if the code later moves to objects[2] for instance, or if we want to decorate 201/203 etc..) it would be much easier to maintain.

(this applies for the 2 lines below as well)

Reply:
:+1: